pkg/ebpf/events_derived.go

```diff
@@ -6,20 +6,16 @@ import (
 	"strings"
 
 	"github.com/aquasecurity/libbpfgo/helpers"
+	"github.com/aquasecurity/tracee/pkg/ebpf/events/derived"
+	"github.com/aquasecurity/tracee/pkg/events/parsing"
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
-// deriveFn is a function prototype for a function that receives an event as
-// argument and may produce a new event if relevant.
-// It returns a derived or empty event, depending on successful derivation,
-// a bool indicating if an event was derived, and an error if one occurred.
-type deriveFn func(trace.Event) (trace.Event, bool, error)
-
 // Initialize the eventDerivations map.
 // Here we declare for each Event (represented through it's ID)
 // to which other Events it can be derived and the corresponding function to derive into that Event.
 func (t *Tracee) initEventDerivationMap() error {
```

Comment:
So I don't see here anywhere where the change to return multiple events from a derivation was used. So i'm not sure what the point of the change was.